validator/sawtooth_validator/execution/processor_manager.py

```diff
@@ -149,6 +149,9 @@ def wait_for_registration(self, processor_type):
             None
         """
         with self._condition:
+            if processor_type not in self:
+                LOGGER.debug("waiting for processor type %s to register",
```

Comment:
Maybe this should be an info level message

Reply:
Or a warning? This is going to harm the normal operation soon.